src/components/modal/v2/parts/Header.jsx

```diff
@@ -68,7 +78,7 @@ const Header = ({ headline, subheadline, className = '', logo, contentWrapper, c
                 </div>
                 <div className="header__content">
                     <h1>{headline}</h1>
-                    <h2>{subheadline}</h2>
+                    {isQualifying === 'true' ? <h2>{qualifyingSubheadline ?? subheadline}</h2> : <h2>{subheadline}</h2>}
```

Comment:
```suggestion
                    {isQualifying === true ? <h2>{qualifyingSubheadline ?? subheadline}</h2> : <h2>{subheadline}</h2>}
```
If we change this to a boolean true instead of a string true, the check works as expected.